paper/author_disambiguation.tex

```diff
@@ -697,7 +665,7 @@ \subsection{Results and discussion}
 As Table~\ref{table:results} shows, switching to either Double metaphone or NYSIIS phonetic-based blocking allows
 to improve the overall F-measure score, trading precision for recall.
 In particular, the NYSIIS-based phonetic blocking shows to be the most effective when applied
-to the baseline (with an F-measure of $0.9850$) while also being the most
+to the state-of-the-art (with an F-measure of $0.9850$) while also being the most
```

Comment:
I dont really like using the name "state-of-the-art" for that combination (here and below). It is misleading in my opinion. CC @natsheh @MSusik